portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,8 +67,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       maxNumComments = Integer.parseInt(maxNumCommentsString);
     } catch (NumberFormatException e) {
       System.err.println("Could not convert to int: " + maxNumCommentsString);
-      System.err.println("Using default value of 7.");
-      maxNumComments = 7;
+      System.err.println("Using default value of " + defaultMaxNumComments + ".");
```

Comment:
No need to change anything now, but when the time comes for logging, we'll want to use Java.util.logging, per the [app engine guide](https://cloud.google.com/appengine/docs/standard/java/logs#writing_application_logs) - just a heads up for the capstone.

Reply:
Ok good to know.